src/app/components/PortraitVideoCarousel/index.styles.tsx

```diff
@@ -0,0 +1,140 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+
+export const PROMO_ITEM_WIDTH = 180; // fixed width for one promo item
+
+const styles = {
+  heading: ({ fontSizes, fontVariants, palette }: Theme) =>
+    css({
+      ...fontVariants.sansBold,
+      ...fontSizes.doublePica,
+      color: palette.GREY_10,
+    }),
+
+  scrollContainer: () =>
+    css({
+      position: 'relative',
+      display: 'flex',
+      alignItems: 'center',
+    }),
+
+  scrollWrapper: ({ spacings }: Theme) =>
+    css({
+      display: 'flex',
+      overflowX: 'auto',
+      scrollSnapType: 'x mandatory',
+      gap: `${spacings.DOUBLE}rem`,
+      scrollBehavior: 'smooth',
+      WebkitOverflowScrolling: 'touch',
+      '&::-webkit-scrollbar': {
+        display: 'none',
+      },
+      scrollbarWidth: 'none',
+      flex: 1,
+    }),
+
+  promoItem: ({ mq }: Theme) =>
+    css({
+      scrollSnapAlign: 'start',
+      flex: '0 0 35%',
+      textDecoration: 'none',
+      display: 'block',
+      position: 'relative',
+      overflow: 'hidden',
+
+      [mq.GROUP_3_MIN_WIDTH]: {
```

Comment:
Added this breakpoint checks in to copy(ish) the way PS hide the navigation below a certain breakpoint.